<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 255, 255)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:40pt; color:rgb(0,0,0)</string>
   </property>
   <property name="text">
    <string>Create A New Account</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: rgb(105, 105, 105);
font: 20pt &quot;Avenir&quot;;</string>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: rgb(105, 105, 105);
font: 20pt &quot;Avenir&quot;;</string>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:17pt; color: rgb(105, 105, 105);
font: 20pt &quot;Avenir&quot;;</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: rgb(105, 105, 105);
font: 20pt &quot;Avenir&quot;;</string>
   </property>
   <property name="text">
    <string>Date of Birth (M/D/Y)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>410</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: rgb(105, 105, 105);
font: 20pt &quot;Avenir&quot;;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fname">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lname">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dob">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signUp">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>580</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0,0,0); font-size:20pt;</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>650</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0,0,0);font-size: 20pt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>470</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: rgb(105, 105, 105);
font: 20pt &quot;Avenir&quot;;</string>
   </property>
   <property name="text">
    <string>Confirm Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="con_password">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>470</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="gobacklogin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>580</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0,0,0); font-size:15pt;</string>
   </property>
   <property name="text">
    <string>Go Back Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>530</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: rgb(105, 105, 105);
font: 20pt &quot;Avenir&quot;;</string>
   </property>
   <property name="text">
    <string>Album name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="album_name_input">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>530</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
